editors/code/package.json

```
@@ -151,6 +151,11 @@
           "scope": "window",
           "default": "off"
         },
+        "luau-lsp.serverPath": {
```

Can we name this `luau-lsp.server.path`?

It allows us to extend the server configuration with other options in future

```suggestion
        "luau-lsp.server.path": {
```